docker-compose.yml

```
@@ -1,9 +1,31 @@
+    tty: true
+
+  hogwartsredis:
+    image: "redis:alpine"
     ports:
-      - "8080:8080"
\ No newline at end of file
+      - "6379:6379"
+    environment:
+      - REDIS_PASSWORD=test
+    command: ["redis-server", "--requirepass", "test"]
+    tty: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)